docs/additional_examples.md

```diff
@@ -0,0 +1,121 @@
+## MDBCI usage
+
+### Generate product repositories
```

Comment:
This part MUST not be present in all the documents that deal with the configuration. Please remove it from this document.